infrastructure/ArmTemplates/DeployAllTemplate.json

```diff
@@ -103,105 +96,128 @@
                     "siteName": "[replace(parameters('RuntimeHost1'),'1','2')]"
                 }
             ]
+        },
+        "RedisCache": {
+            "Name": "LiberationCache-Prod",
```

Comment:
Can we name it something like diagnostics cache or app service diagnostics cache?

Reply:
That sounds better, I'll do DiagnosticsCache